.github/workflows/coverage.yml

```diff
@@ -9,22 +9,7 @@ on:
   pull_request:
 
 jobs:
-  Coveralls:
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [16.x]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npx jest --coverage
-    - name: Coveralls
-      uses: coverallsapp/github-action@master
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+  call_code_coverage:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/coverage.yml@main
```

Comment:
Do you think it would be a good idea to point at a versioned branch rather than main? For example, we could make a branch called 'v1' and push and minor or patch versions to it as needed. That way if we ever want to make breaking changes, we can make a 'v2' branch off of master without breaking the repos which depend on the 'v1' branch

Reply:
Yeah, that makes sense. I'll update